pkgs/applications/networking/cluster/k3s/1_29/versions.nix

```diff
@@ -1,8 +1,8 @@
 {
-  k3sVersion = "1.29.0+k3s1";
-  k3sCommit = "3190a5faa28d7a0d428c756d67adcab7eb11e6a5";
-  k3sRepoSha256 = "1g75a7kz9nnv0vagzhggkw0zqigykimdwsmibgssa8vyjpg7idda";
-  k3sVendorHash = "sha256-iHmPVjYR/ZLH9UZ5yNEApyuGQsEwtxVbQw7Pu7WrpaQ=";
+  k3sVersion = "1.29.1+k3s1";
```

Comment:
Latest should be [1.29.1+k3s2](https://github.com/k3s-io/k3s/releases/tag/v1.29.1%2Bk3s2).